src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const envConnectHost = "OP_CONNECT_HOST";
+export const envConnectToken = "OP_CONNECT_TOKEN";
+export const envServiceAccountToken = "OP_SERVICE_ACCOUNT_TOKEN";
+export const envManagedVariables = "OP_MANAGED_VARIABLES";
```

Comment:
I'm sort of on the fence about this convention because I see it as a relic of a time when we _needed_ to use upper-snake-case to denote constant/immutable values, but we've got `const` now (although you can still an const object's properties but that's a separate gripe) and you'll get a runtime error if you reassign it.

That said, if we do uppercase I don't see your suggestion as being terribly redundant - it still serves its purpose correctly (it's not our fault the value is also uppercase haha).

Reply:
I'd prefer to keep these constants camelCase. For some reason, I feel like it looks better this way (may be because of Go). Also, Writing constant with capitals may lead to more errors? Not sure about that one.

If you feel better about having these constants use the `UPPERCASE_VARIABLES` format, then I'm up for it.

Reply:
I'm OK to leave them as-is in camcelCase - as Jody mentioned before the `const` is enough indication that these are immutable constant values. (Also, we have the `constants.ts` filename as an extra reminder 😄)